deploy/src/main/resources/README.md

```diff
@@ -48,180 +46,28 @@ The Docker Compose file contains the following services:
 
     By default the output folder used is `./output-files`, but a different folder can be used by setting the `JOB_SERVICE_DEMO_OUTPUT_DIR` environment variable.
 
-## Usage
```

Comment:
As discussed, we're losing too much stuff here.  For me, the main purpose of this 'deploy' branch is the quick-start.